docs/administering/sniproxy.md

```diff
@@ -0,0 +1,146 @@
+
+The main reason is to allow users to connect with your Sandstorm instance in the standard HTTPS port (443) and keep using that port also for any other web apps.
+
+This action is going to change the BASE_URL of Sandstorm, presumably, so you'll end up erasing the OAuth configuration. To fix that, you will need to follow the "How do I log in, if there's a problem with logging in via the web?" section of [the FAQ](faq.md).
+
+This document assumes the server is running Debian or one of its derivatives (e.g. Ubuntu). If your server is not, you might need to adjust some steps. Note that there will be some downtime in this process so you might want to do it when your server is not very busy.
+
+## Install sniproxy
+
+If you're lucky, the package `sniproxy` might be present on your GNU/Linux distro; otherwise, you'll have to install it yourself. Follow the instructions at [the sniproxy homepage](https://github.com/dlundquist/sniproxy) for your operating system and install it.
+
+In this example, on Ubuntu 10.04, the instructions were:
+
+```bash
+# Install required packages
+sudo apt-get install autotools-dev cdbs debhelper dh-autoreconf dpkg-dev gettext libev-dev libpcre3-dev libudns-dev pkg-config fakeroot git
+
+# Clone sniproxy repo from Github
+git clone https://github.com/dlundquist/sniproxy.git
+
+# Compile and create the package
+cd sniproxy
+./autogen.sh && dpkg-buildpackage
+
+# Install the package
+sudo dpkg -i ../sniproxy_*_*.deb
+```
+
+## Setting up sniproxy
+
+We'll be using sniproxy to listen on the standard HTTPS port (443) and allow it to decide where to send the request based on the hostname being requested. If it's a Sandstorm domain (ends in `.sandcats.io`) it will forward the request to Sandstorm on port 6443. In any other case it'll forward the request to the web server you already had, which we will switch from listening on port 443 to port 7443.
+
+The ports for Sandstorm and the web server are arbitrary; you can choose ones that work for you in case you have a collision with another service you're running. It should work as long as you're consistent in replacing the ports in the web server, Sandstorm and sniproxy configuration files. We'll also disable the HTTP proxy feature in sniproxy as there is no need for the HTTP requests to go through it.
+
+### Configuration
+
+You will need to edit the sniproxy configuration file by running something like:
+
+```bash
+sudo nano /etc/sniproxy.conf
+```
+
+Here is the example configuration for `/etc/sniproxy.conf`:
+
+```
+# sniproxy.conf
+# Setup for sharing port 443 with Sandstorm
+
+user daemon
+pidfile /var/run/sniproxy.pid
+
+error_log {
+    syslog daemon
+    priority notice
+}
+
+listen 443 {
+    proto tls
+    table https_hosts
+    fallback 127.0.0.1:7443
+
+    access_log {
+        filename /var/log/sniproxy/https_access.log
+        priority notice
+    }
+}
+
+table https_hosts {
+    .*\.sandcats\.io 127.0.0.1:6443
+}
+```
+
+### Startup
```

Comment:
This section also looks bitrotten to me; probably a modern version would be something like `systemctl enable sniproxy`.

Reply:
I think we should probably document both... Sandstorm continues to support sysvinit-based setup, see here: https://github.com/sandstorm-io/sandstorm/blob/master/install.sh#L1610